### DISCUSSION TITLE
Disabling Command log.

### DISCUSSION THREAD

**Discussion by klem95** [cypress-io/cypress#22758] [Questions and Help] [answered]
Hi, I was wondering whether it would be possible to disable ALL prints that is made in the Command log(preferable not showing the log window at all)? Cypress runs inside a “split” window, and I want to disable the prints made in the reporter (in the left sides of the window). The reason behind this, is to ensure that the system is not slowed down by being forced to print commands.

I have tried to create a custom reporter; however, it does not seem to affect the Cypress browser window.

I have also tried CYPRESS_NO_COMMAND_LOG=1, which does not seem to have any effect. I have tried parsing it in both cypress.env.json and cypress.json (I'm using version 6.6.0)

Any suggestions would be much appreciated.

**Comment by stavares843** [comment]
Hello! 👋 

seems there is a PR to fix the above [here](https://github.com/cypress-io/cypress/pull/22641) 🔨

**Reply by klem95** [reply]
Hi @stavares843, thanks for the link! I found out my problem was that the environment variable name: CYPRESS_NO_COMMAND_LOG, was introduced in a later version, and was originally called: NO_COMMAND_LOG. This is the case in the version I'm currently using (6.6.0)

**Comment by klem95** [comment, marked as answer]
I found out my problem was that the environment variable name: CYPRESS_NO_COMMAND_LOG, was introduced in a later version, and was originally called: NO_COMMAND_LOG. This is the case in the version I'm currently using (6.6.0)
